sonar-project.properties

```diff
@@ -1,18 +1,14 @@
-sonar.projectKey=metamask-extension
-sonar.organization=consensys
+sonar.projectKey=metamask-extension-private
+sonar.organization=metamask
```

Comment:
I've seen that in Mobile project, we are using `consensys` as the organization. Is there any particular reason to use `metamask` instead of `consensys` ?

https://github.com/MetaMask/metamask-mobile/blob/main/sonar-project.properties#L2